cocos/core/renderer/core/memory-pools.ts

```diff
@@ -35,7 +35,12 @@ interface TypedArrayConstructor<T> {
     readonly BYTES_PER_ELEMENT: number;
 }
 
-export class NativeBufferPool<T extends TypedArray> {
+class NativeBufferPool {
```

Comment:
Please separate to different file, otherwise it can't be ruled out in native build